### PR TITLE
CC and Kember viewers output csv files

### LINKED ISSUE #645: Adapt dy-viewers to output plot data
Adapt jupyter notebook based viewers to export graph data into cvs files
- [ ] Colleen Clancy 0D
- [ ] Colleen Clancy 1D
- [ ] Colleen Clancy 2D
- [ ] Colleen Clancy Kember

child of #3
child of #4
child of #5

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?

This PR modifies the 3 CC and Kember viewers to provide in their outputs (in CSV format) the data used in the internal plots.
- [x] Adapt services to new output labels
- [x] Add helper function in simcore-sdk package to write csv file for the given pandas dataframe
- [x] Adapt services to wirte the csv files

## Related issue number

closes #645 

## Checklist

- [ ] I think the code is well written
- [ ] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [ ] **Runs in the swarm**
- [ ] If you design a new module, add your user to .github/CODEOWNERS

[waffle.io]:https://waffle.io/marketing-assets/documents/waffleio_cheatsheet_v1.pdf?utm_source=blog&utm_medium=cheatsheet-ctabutton&utm_campaign=cheatsheet

### CODE REVIEW

File: services/dy-2Dgraph/use-cases/cc/cc-oned/notebook/cc_oned.ipynb [hunk 1]
```diff
@@ -170,8 +170,8 @@
     "    data_frame_a = pd.read_csv(data_path_a, sep=\"\\t\", header=None)\n",
     "    # membrane potential\n",
     "    axis_colums = [0,1]\n",
-    "    plot_data = data_frame_a.filter(items=[data_frame_a.columns[i] for i in axis_colums])\n",
-    "    create_graph(data_frame=plot_data)"
+    "    plot_0 = data_frame_a.filter(items=[data_frame_a.columns[i] for i in axis_colums])\n",
+    "    create_graph(data_frame=plot_0)"
```
Comment (sanderegg) [inline services/dy-2Dgraph/use-cases/cc/cc-oned/notebook/cc_oned.ipynb]: don;'t see exactly why you 're doing this... but ok...

File: services/dy-2Dgraph/use-cases/cc/cc-single-cell/notebook/cc_single_cell.ipynb [hunk 2]
```diff
@@ -212,8 +212,8 @@
     "# membrane potential\n",
     "title=\"Membrane Potential\"\n",
     "axis_colums = [0,ynid[39]+1]\n",
-    "plot_data = data_frame_ty.filter(items=[data_frame_ty.columns[i] for i in axis_colums])\n",
-    "create_graph(data_frame=plot_data, \n",
+    "plot_0 = data_frame_ty.filter(items=[data_frame_ty.columns[i] for i in axis_colums])\n",
+    "create_graph(data_frame=plot_0, \n",
```
Comment (sanderegg) [inline services/dy-2Dgraph/use-cases/cc/cc-single-cell/notebook/cc_single_cell.ipynb]: same

File: services/dy-2Dgraph/use-cases/cc/cc-single-cell/notebook/cc_single_cell.ipynb [hunk 3]
```diff
@@ -250,16 +250,19 @@
     "# CaSRT & Caj\n",
     "data_frame_casrt = data_frame_ty.filter(items=[data_frame_ty.columns[0], data_frame_ty.columns[ynid[30]+1], data_frame_ty.columns[ynid[31]+1]])\n",
     "data_frame_casrt[3] = data_frame_casrt[1] + data_frame_casrt[2]\n",
-    "plot_data = [data_frame_casrt.filter(items=[data_frame_casrt.columns[0], data_frame_casrt.columns[3]])]\n",
+    "plot_2 = data_frame_casrt.filter(items=[data_frame_casrt.columns[0], data_frame_casrt.columns[3]])\n",
+    "plot_data = [plot_2]\n",
```
Comment (sanderegg) [inline services/dy-2Dgraph/use-cases/cc/cc-single-cell/notebook/cc_single_cell.ipynb]: why?

File: services/dy-2Dgraph/use-cases/cc/cc-single-cell/notebook/cc_single_cell.ipynb [hunk 4]
```diff
@@ -509,6 +521,39 @@
...
+   "metadata": {
+    "extensions": {},
+    "hide_input": true,
+    "init_cell": true
+   },
+   "outputs": [],
+   "source": [
+    "# Export data to CSV\n",
+    "import csv_adapter\n",
+    "await csv_adapter.pandas_dataframe_to_csv(plot_0, \"Membrane Potential\", [\"time (sec)\", \"Membrane Potential\"], 0)\n",
```
Comment (sanderegg) [inline services/dy-2Dgraph/use-cases/cc/cc-single-cell/notebook/cc_single_cell.ipynb]: for loop?
```python
for plot in plot_data:
   await csv_adapter.pandas....
```

File: services/dy-2Dgraph/use-cases/cc/cc-oned/notebook/cc_oned.ipynb (hunk 1 above)
Comment (odeimaiz) [reply services/dy-2Dgraph/use-cases/cc/cc-oned/notebook/cc_oned.ipynb]: plot_0 is used later on for exporting all the plots to csv

**Pull request closed without merging**